schema/rdl_schema_0.1.json

```
@@ -1025,6 +788,104 @@
 				"Hybrid"
 			]
+		},
+		"Spatial": {
+			"title": "Spatial coverage",
+			"type": "object",
+			"description": "Information about the spatial coverage of the dataset.",
+			"properties": {
+				"countries": {
+					"title": "Countries",
+					"type": "array",
+					"description": "The countries covered by the geographical area, from the open country codelist (ISO 3166-1).",
+					"items": {
+						"type": "string"
+					},
+					"minItems": 1
+				},
+				"gazetteerEntries": {
+					"title": "Gazetteer entries",
+					"type": "array",
+					"description": "Entries from geographical indices or directories describing the geographical area. This field should be used to describe subnational coverage. Use of is recommended.",
+					"items": {
+            "$ref": "#/$defs/Gazetteer_entry"
+          },
+          "uniqueItems": true,
+					"minItems": 1
+				},
+				"bbox": {
+					"title": "Bounding box",
+					"type": "array",
+					"description": "A geographic bounding box delimiting the geographical area.",
+					"items": {
+						"type": "number"
+					},
+					"minItems": 1
+				},
+				"geometry": {
+					"title": "Geometry",
+					"description": "A set of coordinates denoting the vertices of the geographical area.",
+					"$ref": "#/$defs/Geometry"
+				},
+				"centroid": {
+					"title": "Centroid",
+					"type": "array",
+					"description": "The coordinates of the centre of the geographical area.",
+					"items": {
+						"type": "number"
+					},
+					"minItems": 1
+				}
+			}
+		},
+		"Gazetteer_entry": {
+			"title": "Gazetteer entry",
+			"type": "object",
+			"description": "Entry from geographical indices or directories describing the geographical area.",
+			"properties": {
+				"id": {
+					"title": "Identifier",
+					"type": "string",
+					"description": "An identifier drawn from the gazetteer identified in `.scheme`."
+				},
+				"scheme": {
+					"title": "Scheme",
+					"type": "string",
+					"description": "The gazetteer from which the entry is drawn, from the open location_ gazetteers codelist."
+				},
+				"description": {
+					"title": "Description",
+					"type": "string",
+					"description": "A description for the gazetteer entry."
+				},
+				"uri": {
+					"title": "Uniform resource locator",
+					"type": "string",
+					"description": "A URI for the gazetteer entry.",
+					"format": "iri"
+				}
+			}
+		},
+		"Geometry": {
+			"title": "Geometry",
+			"type": "object",
+			"description": "A set of coordinates denoting the vertices of the geographical area.",
+			"properties": {
+				"type": {
+					"title": "Type",
+					"type": "string",
+					"description": "The GeoJSON geometry type that is described by `.coordinates`, from the closed geometry_type codelist."
+				},
+				"coordinates": {
+					"title": "Coordinates",
+					"type": "array",
+					"description": "One or more GeoJSON positions according to the GeoJSON geometry type defined in `.type`.",
+					"items": {
+						"type": "number"
```

Need to support nested arrays for the more complex geometry types.

```suggestion
						"type": [
						  "number",
						  "array"
						]
```